---
/detect_object:
  ros__parameters:
        red:
          hue_l: 7
          hue_h: 170
          saturation_l: 150
          saturation_h: 255
          lightness_l: 50
          lightness_h: 255
        yellow:
          hue_l: 32
          hue_h: 33
          saturation_l: 233
          saturation_h: 255
          lightness_l: 231
          lightness_h: 255
        green:
          hue_l: 112
          hue_h: 113
          saturation_l: 210
          saturation_h: 255
          lightness_l: 131
          lightness_h: 255
